### ISSUE TITLE
Menu still disappears

### ISSUE THREAD

**Issue by FieryFoxtail** [karmeleon/copy-as-png#3]
Hey there,

The 'Copy as PNG' menu still eventually disappears, though I can't replicate it other than through normal usage, which can take days. It comes back when I turn the extension off and on again. I'm on Firefox 120.0 on Windows 10.

**Comment by karmeleon** [comment]
yeah it does, unfortunately. for w/e reason, the ways that MDN lists to register a dropdown don't work. will have to register it on more events :(
